static/app/views/starfish/views/appStartup/screenSummary/startDurationWidget.tsx

```diff
@@ -15,25 +16,25 @@ import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
 import {STARFISH_CHART_INTERVAL_FIDELITY} from 'sentry/views/starfish/utils/constants';
 import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
 import {useEventsStatsQuery} from 'sentry/views/starfish/utils/useEventsStatsQuery';
-import {
-  COLD_START_COLOR,
-  WARM_START_COLOR,
-} from 'sentry/views/starfish/views/appStartup/screenSummary/appStartBreakdownWidget';
-import {OUTPUT_TYPE, YAxis} from 'sentry/views/starfish/views/screens';
 
-const SPAN_OP_TO_STRING = {
-  'app.start.cold': t('Cold Start'),
-  'app.start.warm': t('Warm Start'),
-};
+const COLD_START_CONDITIONS = ['span.op:app.start.cold', 'span.description:"Cold Start"'];
+const WARM_START_CONDITIONS = ['span.op:app.start.warm', 'span.description:"Warm Start"'];
 
-function transformData(data?: MultiSeriesEventsStats) {
-  const transformedSeries: {[yAxisName: string]: Series} = {};
+const PRIMARY_RELEASE_COLOR = '#444674';
+const SECONDARY_RELEASE_COLOR = '#e9626e';
```

Comment:
Same note about `static/app/views/starfish/colours.tsx` if this is going to be used anywhere else, otherwise no worries